### PR TITLE
Fixed bugs on windows (tested in neovide)

### PR DESCRIPTION
**Pull request by green726**
Opening browsers wasn't working on windows (in neovide) - the function to get the OS version returned Windows_NT not Windows, so it wasn't setting the correct start command. To fix this I just added Windows_NT to the if statement. Also, the lua string formatting wasn't working properly and would combine the start command with the search term for input search (example: input "hi", it would run "starthi www.google.com" so I used concatenation instead.

### CODE REVIEW

File: lua/browse/bookmarks.lua [hunk 1]
```diff
@@ -25,7 +25,8 @@ M.search_bookmarks = function(config)
         actions.close(prompt_bufnr)
         local selection = action_state.get_selected_entry()
         local open_cmd = utils.get_open_cmd()
-        vim.fn.jobstart(string.format("%s '%s'", open_cmd, selection[1]))
+        local cmd = open_cmd.." "..selection[1]
```
Comment (lalitmee) [inline lua/browse/bookmarks.lua]: Can you please format the code using `stylua`?

File: lua/browse/input.lua [hunk 2]
```diff
@@ -8,7 +8,9 @@ M.search_input = function()
       return
     end
     local open_cmd = utils.get_open_cmd()
-    vim.fn.jobstart(string.format("%s 'https://www.google.com/search?q=%s'", open_cmd, input))
+    local cmd = open_cmd.." https://www.google.com/search?q="..input
+    vim.cmd(":silent ! "..cmd)
+    -- vim.fn.jobstart(string.format("%s 'https://www.google.com/search?q=%s'", open_cmd, input))
```
Comment (lalitmee) [inline lua/browse/input.lua]: We will have to test this. Why are we doing this with `vim.cmd`?

**Review by lalitmee — CHANGES_REQUESTED** [review]
@green726, thanks for your contribution. I appreciate it a lot.

File: lua/browse/input.lua (hunk 2 above)
Comment (green726) [reply lua/browse/input.lua]: Hello, I was using vim.cmd to test and forgot to revert back to jobstart. I have now done that as well as formatted the code with stylua

**Review by lalitmee — CHANGES_REQUESTED** [review]
You are formatting it with `Tabs`, but we have `spaces`. Can you please do that? If you don't know how to do that then simply read at [stylua config](https://github.com/JohnnyMorganz/StyLua#finding-the-configuration)

**Pull request merged**